ourjapanlife/findadoc-frontend
# Add team leads to about page (#140).

## Resolves #140

This is a basic version of a card-based team-leads listing for the site.
Right now, only Ann has a custom pic; for everyone else it's defaulting back to your github profile.
Also, I don't know Teresa's last name...can someone provide this?

In the future, I think we should add another card that displays a mini-grid of recent contributors to include more of the community, but that can wait for a future PR.

### How to test

1. Stand up dev stack and go to about page
2. Confirm that cards appear for devs near the bottom
3. Switch to mobile emulation and test responsiveness

### Screenshots

<img width="1440" alt="スクリーンショット 2021-06-27 17 28 02" src="https://user-images.githubusercontent.com/6155643/123537991-6290dc80-d76d-11eb-8d9f-cbb38ef467f5.png">
<img width="374" alt="スクリーンショット 2021-06-27 17 29 22" src="https://user-images.githubusercontent.com/6155643/123538018-8b18d680-d76d-11eb-8be0-f155766daf67.png">
